features/networking/sdn.feature

```diff
@@ -220,11 +218,11 @@ Feature: SDN related networking scenarios
     And evaluation of `pod(1).name` is stored in the :pod2_name clipboard
 
     # Kill ovs process
-    When I run command on the "<%= cb.node_name %>" node's sdn pod:
-      | bash | -c | pgrep ovs-vswitchd |
+    When I run command on the "<%= cb.node_name %>" node's ovs pod:
```

Comment:
Ok its changed to pkilling on ovs pods which means this can be now run on both ovn and sdn. @huiran0826 Do you think OCP25706 should target both networktypes now? We might have to change ```And I use the "openshift-sdn" project``` -> ```Given I store node's corresponding default networkType pod name in the clipboard```. Once #1365 merges, we will get correpsonding networktype project name gets stores in ```cb.network_project_name```